doc/contributor/howto-guide-setup-development-workstation.md

```diff
@@ -151,8 +151,8 @@ env -C cmake-out/home \
 ### Installing Docker
 
 You may want to [install Docker](https://docs.docker.com/engine/installation/),
-this will allow you to use the build scripts to test on multiple platforms,
-as described in [CONTRIBUTING.md](../../CONTRIBUTING.md).
+this will allow you to [use the build scripts](howto-guide-running-ci-builds-locally.md)
```

Comment:
"as this ..." perhaps?

Reply:
Good catch. I split it into two sentences.